packages/teams-js/src/public/search.ts

```diff
@@ -0,0 +1,131 @@
+import { sendMessageToParent } from '../internal/communication';
+import { registerHandler, removeHandler } from '../internal/handlers';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Allows you to interact with the host search experience
```

Comment:
You should make sure to work with your PM to think about what documentation you think would be appropriate to put into this file. What I originally put here is correct, but not very descriptive. This comment (and others in this file) will get automatically turned into developer documentation available to 1P and 3P developers and can be a good place to describe what this capability is and why a developer may want to use it.

If it's helpful, I can also put you in contact with some documentation people on our side to help review doc language.

Reply:
I've sent Diana some suggestions for the documentation offline too.